casper/src/main/scala/io/casperlabs/casper/util/ProtoUtil.scala

```diff
@@ -330,6 +330,27 @@ object ProtoUtil {
   def blockNumber(b: Block): Long =
     b.getHeader.rank
 
+  /** Removes redundant messages that are available in the immediate justifications of another message in the set */
+  def removeRedundantMessages(
+      messages: Iterable[Message]
+  ): Set[Message] = {
+    // Builds a dependencies map.
+    // ancestor -> {descendant}
+    // Allows for quick test whether a block is in justifications of another one.
+    val dependenciesOf = messages
```

Comment:
```suggestion
    val dependantsOf = messages
```

Dependencies are used in the other direction throughout the codebase.

Reply:
Done.